BucketeerTests/E2E/BucketeerE2ETests.swift

```diff
@@ -42,8 +42,9 @@ final class BucketeerE2ETests: XCTestCase {
         assertEvaluation(actual: actual, expected: .init(
             id: "feature-ios-e2e-string:2:bucketeer-ios-user-id-1",
             featureId: FEATURE_ID_STRING,
-            featureVersion: 2,
+            featureVersion: 3,
```

Comment:
Because I added the variation name on the console UI, it increased the feature version.
Same for the other tests.